vignettes/EIRprevmatch.Rmd

```diff
@@ -15,68 +15,159 @@ knitr::opts_chunk$set(
+
+## *Pf*PR~2-10~ matching using malariasimulation
+
+Where the `malariaEquilibrium` method is not viable, an alternative is to run `malariasimulation` simulations over a smaller range of initial EIR values, extract the PfPR2-10 from each run, fit a model relating initial EIR to PfPR2-10, and use the model to predict the initial EIR value required to yield the desired PfPR2-10. This approach allows users to benefit from the tremendous flexibility in human population, mosquito population, and intervention package parameters afforded by the `malariasimulation` package. An example of this method is outlined below.
+
+### Establish malariasimulation parameters
+
+To run `malariasimulation`, the first step is to generate a list of parameters using the `get_parameters()` function, which loads the default `malariasimulation` parameter list. Within this function call, we adapt the average age of the human population to flatten its demographic profile, instruct the model to output malaria prevalence in the age range of 2-10 years old, and switch off the individual-based mosquito module. The `set_species()` function is then called to specify a vector community composed of *An. gambiae*, *An. funestus*, and *An arabiensis* at a ratio of 2:1:1. The `set_drugs()` function is used to append the built-in parameters for artemether lumefantrine (AL) and, finally, the `set_clinical_treatment()` function is called to specify a treatment campaign that distributes AL to 45% of the population in the first time step (t = 1).
+
+```{r}
+
+# Specify the time frame over which to simulate and the human population size:
 year <- 365
 human_population <- 5000
 
-params <- get_parameters(list(
+# Use the get_parameters() function to establish a list of simulation parameters:
+simparams <- get_parameters(list(
+  
+  # Set the population size:
   human_population = human_population,
-  average_age = 8453.323,     # to match flat_demog
-  model_seasonality = TRUE,   # assign seasonality
-  g0 = 0.284596,                              
-  g = c(-0.317878, -0.0017527, 0.116455),     
-  h = c(-0.331361, 0.293128, -0.0617547),     
-  prevalence_rendering_min_ages = 2 * year,    # set prev age range
+  
+  # Set the average age (days) of the population:
+  average_age = 8453.323,   
+  
+  # Instruct model to render prevalence in age group 2-10:
+  prevalence_rendering_min_ages = 2 * year,    
   prevalence_rendering_max_ages = 10 * year,
+  
+  # Turn off the individual mosquitoes module:
   individual_mosquitoes = FALSE))
```

Comment:
This is now by default, so can be removed

Reply:
Removed `individual_mosquitoes = FALSE`.